.travis/install.sh

```diff
@@ -75,6 +75,10 @@ else
             tar zxf openssl-$OPENSSL_VERSION_NUMBER.tar.gz
             cd openssl-$OPENSSL_VERSION_NUMBER
             ./config shared no-asm no-ssl2 -fPIC --prefix="$HOME/$OPENSSL_DIR"
+            # modify the shlib version to a unique one to make sure it doesn't load the system one.
```

Comment:
Err and while we're at it, the `it` in this sentence is a bit ambigious, can we expand the pronoun?